app/services/queries/daily_metrics.rb

```diff
@@ -3,7 +3,7 @@ class DailyMetrics < BaseQueryMetric
     FROM = 14.days
     INTERVAL = 'daily'.freeze
 
-    def initialize(project_id, metric_name)
+    def initialize(project_id:, metric_name:)
```

Comment:
The constructor signature for both the daily and weekly queries went back and forth between keyword args, to positional args, to keyword args again. 😆 Nevertheless, we do need keyword args in this case, right? 🙌

Reply:
jajaj yes, this was because of the changes that came from develop, this ticket has work from hernan too, so we had to merge our changes 😄

Reply:
Awesome, thanks!